packages/core/getAggregatedAvailability/index.ts

```diff
@@ -5,18 +5,28 @@ import { SchedulingType } from "@calcom/prisma/enums";
 import { mergeOverlappingDateRanges } from "./date-range-utils/mergeOverlappingDateRanges";
 
 export const getAggregatedAvailability = (
-  userAvailability: { dateRanges: DateRange[]; user?: { isFixed?: boolean } }[],
+  userAvailability: {
+    dateRanges: DateRange[];
+    dateRangesWithoutOOO: DateRange[];
+    user?: { isFixed?: boolean };
+  }[],
   schedulingType: SchedulingType | null
 ): DateRange[] => {
+  const isTeamEvent =
```

Comment:
for team events we want to use the date ranges that already take into account OOO days